docker-compose/7.2.N-docker-compose.yml

```diff
@@ -157,7 +156,7 @@ services:
       BASE_PATH: ./
       APP_BASE_SHARE_URL: "http://localhost:8080/workspace/#/preview/s"
   control-center:
-    image: quay.io/alfresco/alfresco-admin-app:7.6.1
+    image: quay.io/alfresco/alfresco-admin-app:7.8.1
```

Comment:
is this ok? can't find this component at https://docs.alfresco.com/content-services/7.2/support/